src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/upstart/start-template

```diff
@@ -27,7 +27,11 @@ end script
 # set the working directory of the job processes
 chdir ${{chdir}}
 
+setuid ${{daemon_user}}
```

Comment:
Just curious now if we are executing without sudo, unless I am wrong I think the command should be sudo bin/${{exec}}

I am seeing issues with the daemon_user not having enough permissions